lightning_hpo/commands/sweep/run.py

```diff
@@ -396,6 +410,9 @@ def run(self) -> None:
         repo.package()
         repo.upload(url=f"{URL}/api/v1/upload_file/{name}")
 
+        data_split = [data.split(":") for data in hparams.data]
```

Comment:
We should support not provided a mount_path no ?

Reply:
It is supported 😃 We only take the second part of the split if there is one